ios/COVIDSafePaths/AppDelegate.m

```diff
@@ -80,6 +81,27 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
 }
 
 - (void)applicationWillTerminate:(UIApplication *)application {
+  //Show local notifiation at first time only.
+  if(![[NSUserDefaults standardUserDefaults] boolForKey:@"NotificationAtFirstTimeOnly"]) {
+    if ([[UIApplication sharedApplication] respondsToSelector:@selector(currentUserNotificationSettings)]){
+      //Checking local notification permission or not.
+      UIUserNotificationSettings *grantedSettings = [[UIApplication sharedApplication] currentUserNotificationSettings];
+      if (grantedSettings.types != UIUserNotificationTypeNone){
+        // Set local notification.
+        UILocalNotification *_localNotification = [[UILocalNotification alloc] init];
+        _localNotification.fireDate = [NSDate dateWithTimeIntervalSinceNow:5];
+        _localNotification.timeZone = [NSTimeZone defaultTimeZone];
+        _localNotification.alertTitle = @"COVID Safe Paths";
+        _localNotification.alertBody = @"COVID Safe Paths is securely storing your GPS coordinates once every five minutes on this device.";
+        _localNotification.soundName = UILocalNotificationDefaultSoundName;
+        [[UIApplication sharedApplication]scheduleLocalNotification:_localNotification];
+      }
+    }
+    [[NSUserDefaults standardUserDefaults] setBool:YES forKey:@"NotificationAtFirstTimeOnly"];
+    [[NSUserDefaults standardUserDefaults] synchronize];
+  }
+    
+  //  ***********************************************************************************//
```

Comment:
is the code below doing a similar thing? can we remove it? @tremblerz

Reply:
Mean, to remove from line number 105 to 113?

Reply:
Yes. I think so.


When does that notification show?

Reply:
@tremblerz Plz confirm. Shall I remove from line number 105 to 113?

Reply:
@tstirrat I think that notification show when app quit [Code line number 105 to 113]. It is not working in my iPhone.

Reply:
Yes, please remove the other notification, it's trying to do the same thing.